### PR TITLE
Remove extra nodes

### PR DESCRIPTION
**Pull request by etiennesegonzac**

### CODE REVIEW

**Comment by etiennesegonzac** [comment]
Will rebase on top of #1

**Comment by wilsonpage** [comment]
Wondering if it might be more optimal to do this using a regex when the template is still a string?

**Comment by etiennesegonzac** [comment]
Maybe... I'd like to do it as an enhancement once we have tests.
I'm sure I'm going to screw things up otherwise :)

File: poplar.js [hunk 1]
```diff
@@ -123,6 +124,26 @@ function getDeep(item, parts) {
   return parts.length ? getDeep(item[part], parts) : item[part];
 }
 
+function rocify(el) {
```
Comment (wilsonpage) [inline poplar.js:132]: JSDoc: "Removes empty text nodes recursively to optimize rendering performance"

**Pull request merged**